serverless/sql-databases/api-cli/secure-connection-ssl-tls.mdx

```diff
@@ -0,0 +1,171 @@
+---
+meta:
+  title: Secure connections using SSL/TLS
+  description: This page explains how to configure SSL/TLS to encrypt traffic between your client and Serverless SQL Databases
+content:
+  h1: Secure connections using SSL/TLS
+  paragraph: This page explains how to configure SSL/TLS to encrypt traffic between your client and Serverless SQL Databases
+dates:
+  validation: 2024-06-03
+  posted: 2024-06-03
+---
+
+This documentation will guide you through SSL/TLS configuration in your PostgreSQL-compatible client to ensure your traffic with Scaleway's Serverless SQL Databases is encrypted.
+
+Configuration examples for languages, frameworks and tools: 
+  - [Python](#pythonpsycopg2)
+  - [Python](#pythondjango)
+  - [Node.js](#nodejsnode-postgres)
+  - [Node.js](#nodejspostgresjs)
+  - [Node.js](#javajdbc)
+  - [psql](#psql)
+
+## Generic configuration settings
+
+Starting from PostgreSQL 16, you can set up SSL/TLS to rely on the default certification authority certificates trusted by your operating system. To do so, use the additional configuration parameters `sslmode=verify-full` and `sslrootcert=system`. 
```

Comment:
Should this really be 'certification authority certificates'?

Reply:
![image](https://github.com/scaleway/docs-content/assets/133101456/0ddc674c-e4dc-4c44-ad62-5aac6bbbbb29)
yes 😂

Reply:
or "certificate authority", both exist (I know it sounds weird)